schema/schemaspec/schemahcl/hcl.go

```diff
@@ -72,12 +72,71 @@ func (s *State) Eval(data []byte, v interface{}, input map[string]string) error
 	if err != nil {
 		return fmt.Errorf("schemahcl: failed decoding: %w", err)
 	}
+	if err := patchRefs(spec); err != nil {
+		return err
+	}
```

Comment:
I don't see any test case for this. Is it intentional?

Reply:
it's tested in the drivers, but I will add a specific case in the `schemahcl` package, you're right.